requirements.txt

```diff
@@ -74,3 +74,4 @@ nvidia-cutlass-dsl==4.2.1; python_version >= "3.10"
 plotly
 numexpr<2.14.0 # WAR for attempted use of nonexistent numpy.typing
 partial_json_parser
+tqdm
```

Comment:
we should probably not add requirements here if it's only used in examples. Instead you can provide instructions in the examples to pip install this